include/wav64.h

```diff
@@ -32,8 +33,9 @@ typedef struct {
 	 */
 	waveform_t wave;
 
-	/** @brief Absolute ROM address of WAV64 */
-	uint32_t rom_addr;
+	/** @brief File descriptor to read WAV64 */
+	int currentFile;
```

Comment:
We don't use camelcase in generale, so this should be `current_file` or better `current_fd` or even just `fd` since there's not a "previous" or "next" one to justify calling this "current".